src/gameHandlerClass.ts

```diff
@@ -35,4 +35,15 @@ export class GameHandler {
       ? this._playerTwo
       : this._playerOne;
   }
+
+  canUseMango(): Boolean {
```

Comment:
Why did you add this to game handler? Wee need it only in our player class (we shouldn't put it here because it's a poperty and a method of a player, one player may have a mango, but the second one not necessarily). The whole logic about using mango and healing should be put in the useMango function in playerClass